manifests/plugin/ansible/params.pp

```diff
@@ -12,16 +12,21 @@
   $roles_path = ['/etc/ansible/roles', '/usr/share/ansible/roles']
   $ssh_args = '-o ProxyCommand=none -C -o ControlMaster=auto -o ControlPersist=60s'
   $install_runner = true
-  $manage_runner_repo = true
   case $facts['os']['family'] {
     'RedHat': {
       $callback = 'theforeman.foreman.foreman'
+      $manage_runner_repo = true
+      $runner_package_name = 'ansible-runner'
     }
     'Debian': {
       $callback = 'foreman'
```

Comment:
Unrelated, but isn't Ansible on Focal new enough for the callback?

Reply:
It is. I was just too lazy to add the "if focal" in my last PR *whistle*

Reply:
#667